timeseries/tests/test_SimulatedTimeSeries.py

```diff
@@ -1,66 +1,142 @@
-from SimulatedTimeSeries import SimulatedTimeSeries
+""" test code for SimulatedTimeSeries
```

Comment:
I am not quite sure whether the doc should be put ahead of imports or not. Double check this format (or use pylint)